source/fundamentals/data-formats/documents.txt

```diff
@@ -351,6 +351,54 @@ The output should look something like this:
 
    query result in extended json format: {"_id": {"$oid": "6035210f35bd203721c3eab8"}, "name": "Gabriel García Márquez", "dateOfDeath": {"$date": "2014-04-17T04:00:00Z"}, "novels": [{"title": "One Hundred Years of Solitude", "yearPublished": 1967}, {"title": "Chronicle of a Death Foretold", "yearPublished": 1981}, {"title": "Love in the Time of Cholera", "yearPublished": 1985}]}
 
+.. example::
+   
+   If you would like to work with other formats of JSON strings
+   in your application, you can use the ``JsonObjectCodec`` class
+   along with ``JsonWriterSettings`` to specify your desired JSON
+   format.
+   
+   Below is a code example where where we read and write
+   to our MongoDB instance using 
+   :manual:`Relaxed mode JSON </reference/mongodb-extended-json/>`
```

Comment:
and output ObjectId instances as hex strings

Reply:
Thanks for the feedback Jeff. Good catch. Added  the line here: https://github.com/mongodb/docs-java/pull/57/commits/df9f4cbb9040b62d44a9e48c894e8af4dc99bc40